module/wa/module.py

```diff
@@ -967,18 +968,44 @@ def import_wa(i):
             pname=None
             imported_params={}
 
-            for name, obj in inspect.getmembers(cs):
-                if name!='AndroidUxPerfWorkload' and inspect.isclass(obj):
-                    try:
-                        addr=getattr(cs, name)
-                        pp=addr.parameters
-                        pname=addr.package
-                    except Exception as e:
-                        pass
-
-                    if pp!=None:
-                        wa_name=name
-                        break
+            # Only examine classes
+            for name, obj in inspect.getmembers(cs, predicate=inspect.isclass):
```

Comment:
Ah, sorry I misunderstood. I thought you meant use the ExtensionLoader to make the resource getter available. Not really used to how github UI works, I interpreted it as a comment on the last patch in the series.

I'll see if that works.